packages/core/misc-widgets/src/components/github-star/GithubStar.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div
+    v-if="url"
+    class="kong-ui-public-misc-widgets-github-star"
+    data-testid="github-star"
+  >
+    <a
+      aria-label="i18n.t('githubStar.ariaLabel')"
+      class="github-button"
+      data-color-scheme="no-preference: light; light: light; dark: light;"
+      data-show-count="true"
+      :href="url"
+      target="_blank"
+    >{{ i18n.t('githubStar.title') }}</a>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { onMounted } from 'vue'
+import composables from '../../composables'
+
+defineProps({
+  url: {
+    type: String,
+    required: true,
+  },
+})
+
+const { i18n } = composables.useI18n()
+
+onMounted(async () => {
+  const githubStarScript = document.createElement('script')
+  githubStarScript.setAttribute('src', 'https://buttons.github.io/buttons.js')
+  document.head.appendChild(githubStarScript)
```

Comment:
what happens when the script could not be loaded? Are we still showing component?

Reply:
I thought about that too -- you could have an `onLoad` event bound to the script tag and when called, it would set a variable that allows the component to render in the `v-if`

Reply:
Technically it would still render a valid anchor tag though; just unstyled

Reply:
Good point, checking if I can conditionally display the GHStar only if the script `src` attribute exists

Reply:
True, tried out @adamdehaven's  solution, and it renders the GHStar only if the script is loaded.

<img width="436" alt="Screenshot 2023-02-15 at 12 26 40 PM" src="https://user-images.githubusercontent.com/2568272/219146356-e00aaeb2-af5f-45b5-b477-5a889c50e596.png">